schemas/latest/landingzones/types/backupRecoveryVault.json

```diff
@@ -0,0 +1,53 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "definitions": {
+      "RecoveryVault": {
+        "type": "object",
+        "additionalProperties": false,
+        "properties": {
+          "value": {
+            "oneOf": [
+              {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "enableBackUpRecoveryVault": {
+                    "type": "boolean",
+                    "enum": [
+                      true
+                    ]
+                  },
+                  "name": {
+                    "type": "string"
+                  }
+                },
+                "required": [
+                "enableBackUpRecoveryVault",
```

Comment:
Let's rename `enableBackUpRecoveryVault` to `enabled` so that it's aligned to other resources like sqldb/sqlmi.